commit 93cf21d1496f0afe67d89cbf870b22c27bb713bf (refs/changes/94/1494/2)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2017-02-22T12:17:35-05:00 (2 years, 7 months ago)
    
    OS-5955 make check should fail on cstyle
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/src/Makefile b/src/Makefile
index acd26880..bd81a5fa 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -540,6 +540,7 @@ check: $(JSLINT) $(SUBDIRS)
 	@(for file in $(C_CHECK_TARGETS); do \
 		echo $(PWD)/$$file; \
 		$(CSTYLE) $$file; \
+		[[ $$? == "0" ]] || exit 1; \
 	done)
 	@printf "\nC files ok!\n"
 	@printf "\n==> Running JavaScriptLint...\n"
